.travis.yml

```diff
@@ -22,3 +22,4 @@ addons:
   apt:
     packages:
       - python-sphinx
+      - gdal-bin
```

Comment:
Thanks @jespersm. Well spotted. Two things:
1. GDAL data stores, such as those used in the test, should not need all of `gdal-bin`, just its dependency `libgdal1h`. Please amend this pull request to change `gdal-bin` to `libgdal1h`.
1. Please also change the commit and PR title to "Enable GDAL on Travis". Try not. Do, or do not. There is no try.  ;-)

Reply:
Done!